pom.xml

```diff
@@ -531,7 +531,7 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-surefire-plugin</artifactId>
-					<version>2.18.1</version>
+					<version>3.0.0-M3</version>
```

Comment:
Is this change required as part of the ticket?

Reply:
@dkayiwa initially i was getting test failures in the surefire, failing to execute the maven goal in the plugin untill i decided to upgrade it to the latest version as seen here 
https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-surefire-plugin/3.0.0-M3

Reply:
This change is not related to the ticket and hence should not be in this pull request.